mithril-common/src/certificate_chain/certificate_verifier.rs

```diff
@@ -178,10 +166,14 @@ impl MithrilCertificateVerifier {
         )?;
         let previous_certificate = certificate_retriever
             .get_certificate_details(&certificate.previous_hash)
-            .await?;
+            .await
+            .map_err(|e| e.0)
```

Comment:
Maybe this is possible?
```suggestion
            .map_err(|e| e.into())
```

Reply:
I've already tried it but doesn't seem possible

![image](https://github.com/input-output-hk/mithril/assets/135982616/c29c1706-9348-4bd6-bd7d-f7393a1f8968)

Reply:
Is it better if i had this conversion:

![image](https://github.com/input-output-hk/mithril/assets/135982616/7ac08d74-a8cc-45d6-b2b8-a761fc36361f)

and:
![image](https://github.com/input-output-hk/mithril/assets/135982616/3ff7984e-b1ec-402d-b10e-eb0f25018e21)